meta/documents/user_guide/en/initial_setup.md

```diff
@@ -0,0 +1,119 @@
+
+4. In the Wayfair row of the Plugin set, click on the `Settings` button.
+
+4. In the left-side menu, go to `Configuration` >> `Global Settings`.
+
+5. In the `Supplier Settings` area, enter the `Client ID` and `Client Secret` values that correspond with your Wayfair API credentials
+
+6. Change the `Mode` setting to `Live` - see [information on `Test` mode](test_mode.md)
+
+7. Click the `Save` button in the toolbar above the settings
+
+## 2. Activating the order referrer
+An order referrer in Plentymarkets denotes the sales channel on which an order was generated. To get the Plentymarkets system to properly import orders from the Wayfair API, the Wayfair order referrer must be activated:
+
+1. From the main Plentymarkets page, go to `Setup` >> `Orders` >> `Order referrer`.
+
+2. Place a check mark next to the **Wayfair** order referrer.
+
+3. Click on **Save**.
+
+## 3. Setting up Plentymarkets for shipping through Wayfair
+To ensure proper integrations with Wayfair when shipping order items, follow the procedures outlined in  [the Wayfair Shipping instructions](wayfair_shipping.md).
+
+## 5. Matching items ordered on Wayfair with Item Variations in Plentymarkets:
+In order to properly handle incoming orders from Wayfair, the Wayfair plugin must match the Supplier Part Numbers in Wayfair's systems with a specific field of Item Variations in Plentymarkets. By default, the Wayfair plugin operates on the assumption that the `Variation Number` **(not to be confused with the Variation's ID)** of an Item's Variation in Plentymarkets will match the Wayfair Supplier Part Number.
+
+If the Wayfair Supplier Part Numbers for your organization are to be reflected in an alternative field in your Plentymarkets Item Variations, change the value of [the `Item Mapping Method` setting](settings_guide.md#item-mapping-method) and update the Variations accordingly.
+
+## 6. Making items available for sale on Wayfair
+Items that you want to sell on the Wayfair market must be considered active in Plentymarkets. The Plentymarkets user may also choose limit which Items are for sale on Wayfair. **Note that Inventory and ordered items are controlled at the `Variation` level.**
+
+To ensure that an Item is available for sale, follow these instructions:
+
+1. From the main Plentymarkets page, go to `Item` >> `Edit item`
+
+2. Search for item(s) and open them
+
+3. **For each item**, click `Variations` and open them
+
+4. **For each Variation**:
+
+    1. On the `Settings` tab, make sure that the `Active` checkbox in the `Availability Section`is checked.
+
+    2. If [the `Send all inventory items to Wayfair?` setting](settings_guide.md#send-all-inventory-items-to-wayfair) is **disabled**, go to the `Availability` tab of the Variation and add "Wayfair" to the list in the `Markets` area.
+
+    3. Click the `Save` button next to the Variation `ID` (not the higher-up button for the Item).
+
+
+## 7. Configuring the Warehouse mappings to match Wayfair Supplier IDs.

```

Comment:
The text contains a link to the instructions for doing so. It purposely links out so that we can keep our documentation on the settings match the UX of the settings without having to also come in here and try to explain it for a second time.